debian/control

```diff
@@ -11,6 +11,7 @@ Build-Depends: debhelper (>= 9),
                libicu-dev,
                libreadline-dev,
                gperf,
+               python,
```

Comment:
Why part of clickhouse requires python (apart from tests, whom requires it explicitly)?

Reply:
build deps?